<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fullDialog</class>
 <widget class="QDialog" name="fullDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="disWidget" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>281</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QWidget" name="ctrlWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>401</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/图片/浮动栏/base.png);</string>
   </property>
   <widget class="QPushButton" name="lastButton">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>10</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/图片/浮动栏/last.png);
}
QPushButton:hover{
border-image: url(:/图片/浮动栏/last_pressed.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseButton">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>10</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/图片/浮动栏/pause.png);
}
QPushButton:hover{
border-image: url(:/图片/浮动栏/pause_pressed.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>10</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/图片/浮动栏/play.png);
}
QPushButton:hover{
border-image: url(:/图片/浮动栏/play_pressed.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshButton">
    <property name="geometry">
     <rect>
      <x>268</x>
      <y>10</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/图片/浮动栏/refresh.png);
}
QPushButton:hover{
border-image: url(:/图片/浮动栏/refresh_pressed.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="fullButton">
    <property name="geometry">
     <rect>
      <x>331</x>
      <y>10</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/图片/浮动栏/fullScreen.png);
}
QPushButton:hover{
border-image: url(:/图片/浮动栏/fullScreen_pressed.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>10</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/图片/浮动栏/next.png);
}
QPushButton:hover{
border-image: url(:/图片/浮动栏/next_pressed.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
